packages/opds-web-client/src/components/Book.tsx

```diff
@@ -51,6 +51,7 @@ export default class Book<P extends BookProps> extends React.Component<P, void>
                 <div className="series">{this.props.book.series.name}</div>
               }
               <div className="authors">
+                <span>By </span>
```

Comment:
This probably shouldn’t show if there are no authors and no contributors.

Reply:
Updated.